.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: Testing
+
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
```

Comment:
This is fine for now but is it straightforward to extend for linux 18.04 as well. If so, can we add this?

Reply:
When I checked on April 7th,  Ubuntu-18.04 was available 

```
  build:
    strategy:
      matrix:
        os:
          - "ubuntu-18.04"
          - "ubuntu-22.04"
```

However, Ubuntu-18.04 is soon to enter its Extended Security Maintenance period:

https://ubuntu.com/about/release-cycle

Furthermore, GitHub Actions has declared Ubuntu-18.04 to be deprecated and will become unusable in May 2023. Although it is currently usable, it will likely become unusable soon:

https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/

Does Evolution Gym need to support running on Ubuntu-18.04 going forward? Maintaining an unsupported OS can be difficult. If this support is needed, using GitHub Actions may not be the best option.

Also, I currently believe that maintaining test runs on Ubuntu-22.04 and Ubuntu-latest is more beneficial and easier


```
  build:
    strategy:
      matrix:
        os:
          - "ubuntu-22.04"
          - "ubuntu-latest"

```

Reply:
Thanks for this! `18.04` is where we ran the original experiments, so it would have been nice to support that for reproducibility's sake. I agree `22.04` and `latest` is sufficient for these tests.

Reply:
I will fix configuration the test to ubuntu 22.04 and latest version.

Reply:
https://github.com/EvolutionGym/evogym/pull/25/commits/76b085af2cdca6f41d8967560a199b16ce6e686d